content/en/docs/releases/next-release-preview/index.md

```diff
@@ -39,6 +39,31 @@ controller:
 #### Orca
 Updates Orca's SavePipelineTask to support bulk saves using the updated functionality in the front50 bulk save endpoint.
 
+Keys from the stage context's outputs section can now be removed (there by reducing the context size significantly). 
```

Comment:
This feature only supports these specific tasks, right?

Reply:
That is correct.